src/config.rs

```diff
@@ -19,7 +20,7 @@ pub const OLD_FILE_EXT: &str = "old";
 pub const USER_AGENT: &str = concat!(crate_name!(), "/", crate_version!());
```

Comment:
The USER_AGENT string no longer seems to be used anywhere in this new version of krill-sync?

Reply:
forgot.. added to request headers now.